### PR TITLE
updating the slack invite link

### PR DESCRIPTION
**Pull request by clcoffey**

### CODE REVIEW

File: content/en/events/isc-2021.md [hunk 1]
```diff
@@ -721,4 +721,4 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 All related conversations in the InnerSource Slack Channel will still be considered to fall under [Chatham House Rules](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed can be shared but not attributed.
 
-Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```
Comment (yuhattor) [inline content/en/events/isc-2021.md]:
```suggestion
Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

File: content/en/events/isc-2022.md [hunk 2]
```diff
@@ -767,4 +767,4 @@ The InnerSource Commons ([InnerSourceCommons.org](http://innersourcecommons.org)
 
 All participants, vendors, and guests at InnerSource Commons events are required to abide by the [code of conduct](/about/codeofconduct/).
 All related conversations in the InnerSource Slack Channel will still be considered to fall under [Chatham House Rules](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed can be shared but not attributed.
-Interested in getting involved? Email [summit@innersourcecommons.org](mailto:summit@innersourcecommons.org) or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in getting involved? Email [summit@innersourcecommons.org](mailto:summit@innersourcecommons.org) or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```
Comment (yuhattor) [inline content/en/events/isc-2022.md]:
```suggestion
Interested in getting involved? Email [summit@innersourcecommons.org](mailto:summit@innersourcecommons.org) or join the conversation on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

File: content/en/events/isc-apac-dec-2020.md [hunk 3]
```diff
@@ -59,4 +59,4 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 All related conversations in the InnerSource Slack Channel will still be considered to fall under [Chatham House Rules](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed can be shared but not attributed.
 
-Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```
Comment (yuhattor) [inline content/en/events/isc-apac-dec-2020.md]:
```suggestion
Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

File: content/en/events/isc-fall-2016.md [hunk 4]
```diff
@@ -96,4 +96,4 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 InnerSource Commons meetings run under the [Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed at a meeting can be shared but not attributed.
 
-Interested in helping out? Email <summit@innersourcecommons.org> or speak up on [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw)!
+Interested in helping out? Email <summit@innersourcecommons.org> or speak up on [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA)!
```
Comment (yuhattor) [inline content/en/events/isc-fall-2016.md]:
```suggestion
Interested in helping out? Email <summit@innersourcecommons.org> or speak up on [#innersourcecommons](https://innersourcecommons.org/slack)!
```

File: content/en/events/isc-fall-2017.md [hunk 5]
```diff
@@ -144,7 +144,7 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 InnerSource Commons meetings run under the [Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed at a meeting can be shared but not attributed.
 
-Interested in helping out? Email <summit@innersourcecommons.org> or speak up on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in helping out? Email <summit@innersourcecommons.org> or speak up on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```
Comment (yuhattor) [inline content/en/events/isc-fall-2017.md]:
```suggestion
Interested in helping out? Email <summit@innersourcecommons.org> or speak up on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

File: content/en/events/isc-fall-2018.md [hunk 6]
```diff
@@ -604,5 +604,5 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 InnerSource Commons meetings and events run under the [Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed at a meeting can be shared but not attributed.
 
-Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```
Comment (yuhattor) [inline content/en/events/isc-fall-2018.md]:
```suggestion
Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

File: content/en/events/isc-fall-2020.md [hunk 7]
```diff
@@ -603,4 +603,4 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 All related conversations in the InnerSource Slack Channel will still be considered to fall under [Chatham House Rules](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed can be shared but not attributed.
 
-Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```
Comment (yuhattor) [inline content/en/events/isc-fall-2020.md]:
```suggestion
Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

File: content/en/events/isc-spring-2017.md [hunk 8]
```diff
@@ -29,4 +29,4 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 
 InnerSource Commons meetings run under the [Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed at a meeting can be shared but not attributed.
-https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw)!
+https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA)!
```
Comment (yuhattor) [inline content/en/events/isc-spring-2017.md]:
```suggestion
https://innersourcecommons.org/slack)!
```

File: content/en/events/isc-spring-2018.md [hunk 9]
```diff
@@ -803,7 +803,7 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 InnerSource Commons meetings run under the [Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed at a meeting can be shared but not attributed.
 
-Interested in helping out? Email <summit@innersourcecommons.org> or speak up on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in helping out? Email <summit@innersourcecommons.org> or speak up on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```
Comment (yuhattor) [inline content/en/events/isc-spring-2018.md]:
```suggestion
Interested in helping out? Email <summit@innersourcecommons.org> or speak up on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

File: content/en/events/isc-spring-2019.md [hunk 10]
```diff
@@ -642,7 +642,7 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 InnerSource Commons meetings and events run under the [Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed at a meeting can be shared but not attributed.
 
-Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```
Comment (yuhattor) [inline content/en/events/isc-spring-2019.md]:
```suggestion
Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

File: content/en/events/isc-spring-2020.md [hunk 11]
```diff
@@ -501,4 +501,4 @@ All participants, vendors, and guests at InnerSource Commons events are required
 
 InnerSource Commons meetings and events run under the [Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_Rule): information discussed at a meeting can be shared but not attributed.
 
-Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1l4a602b6-DKD0B0Y8~WE4aDf~o1xCRw) slack channel!
+Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://join.slack.com/t/innersourcecommons/shared_invite/zt-1msf8vcqu-fYEHcyI1l4eSPq6rGprMXA) slack channel!
```
Comment (yuhattor) [inline content/en/events/isc-spring-2020.md]:
```suggestion
Interested in getting involved? Email <summit@innersourcecommons.org> or join the conversation on the [#innersourcecommons](https://innersourcecommons.org/slack) slack channel!
```

**Review by yuhattor — COMMENTED** [review]
By changing it this way, it is possible to respond to all links by simply changing the redirect parameter in [slack.md](https://github.com/InnerSourceCommons/innersourcecommons.org/blob/master/content/en/redirects/slack.md). What do you think?

File: content/en/events/isc-2021.md (hunk 1 above)
Comment (claredillon) [reply content/en/events/isc-2021.md]: Changes suggested by @yuhattor (here and those below) helps address https://github.com/InnerSourceCommons/innersourcecommons.org/issues/454 and looks good to me - but not sure if clicking "commit suggestion" does that to Cristina's branch or the main website - so will leave someone else to do that. Thanks!

Comment (claredillon) [reply content/en/events/isc-2021.md]: Thanks so much @clcoffey for making these changes.

**Review by yuhattor — APPROVED** [review]
The O'Reilly link appears to be broken, and it looks like I need to register with safari books in the first place. In other words, it's not free anymore, is it?
If so, the text needs to be changed. Since this is a PR about the slack invite link, I'll ignore the other link checks and merge them in.
Broken links will be included in a separate PR.

**Pull request merged**